content/docs/setup/kubernetes/upgrade/notice/index.md

```diff
@@ -1,36 +1,41 @@
-- The `ingress` series of options for configuring a Kubernetes Ingress have been removed.  Kubernetes Ingress is still functional and can be enabled using the `--set global.k8sIngress.enabled=true` flag.  Check out the [Securing Kubernetes Ingress with Cert-Manager](/docs/tasks/traffic-management/ingress/ingress-certmgr/) for how to secure your Kubernetes ingress resources.
-
-## Traffic Management
-
-- Outbound traffic policy now defaults to `ALLOW_ANY`.  Traffic to unknown ports will be forwarded as-is. Traffic to known ports (e.g., port 80) will be matched with one of the services in the system and forwarded accordingly.
-- During sidecar routing to a service, destination rules for the target service in the same namespace as the sidecar will take precedence, followed by destination rules in the service’s namespace, and finally followed by destination rules in other namespaces if applicable.
-- We recommend storing gateway resources in the same namespace as the gateway workload (e.g., `istio-system` in case of `istio-ingressgateway`).  When referring to gateway resources in virtual services, use the namespace/name format instead of using `name.namespace.svc.cluster.local`.
-- The optional egress gateway is now disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles by default.  If you need to control and secure your outbound traffic through the egress gateway, you will need to enable `gateways.istio-egressgateway.enabled=true` manually in any of the non-demo profiles.
-
-## Policy & Telemetry
-
-- `istio-policy` check is now disabled by default.  It is enabled in the demo profile for users to explore but disabled in all other profiles.  This change is only for `istio-policy` and not for `istio-telemetry`.  In order to re-enable policy checking, run `helm template` with `--set global.disablePolicyChecks=false` and re-apply the configuration.
-- The Service Graph component has now been deprecated in favor of [Kiali](https://www.kiali.io/).
-
-## Security
-
-- RBAC configuration has been modified to implement cluster scoping.  The `RbacConfig` resource has been replaced with the `ClusterRbacConfig` resource. Refer to [Migrating `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/steps/#migrating-from-rbacconfig-to-clusterrbacconfig) for migration instructions.
+This page describes changes you need to be aware of when upgrading from
+Istio 1.1 to 1.2.  Here, we detail cases where we intentionally broke backwards
+compatibility.  We also mention cases where backwards compatibility was
+preserved but new behavior was introduced that would be surprising to someone
+familiar with the use and operation of Istio 1.1.
+
+For an overview of new features introduced with Istio 1.2, please refer
+to the [1.2 release notes](/about/notes/1.2/).
+
+## Installation and Upgrade
+
+{{< tip >}}
+The vast array of Mixer plugins were deprecated in Istio 1.1.  Please move
```

Comment:
I know this is late (it was impossible to keep up with the constant changes to all of the docs), but i have concerns over coupling adapter deprecation with the configuration model. I also have concerns around the use of the word `plugins`.

IMHO, this warning should be restricted to configuration only. Discussion of deprecating adapters is reasonable, but deserves a lot more context and explanation (to match the config discussion below).

```
The configuration model for Mixer has been simplified. Support for adapter-specific and template-specific Custom Resources has been removed by default in 1.2 and will be removed entirely in 1.3. 
Please move to the new configuration model.
```

@geeknoid thoughts?

Reply:
I was also surprised by the phrasing. I don't think we definitely the term plugin in our docs, which makes the statement ambiguous.

Reply:
@douglas-reid I will submit a PR to fix the warning now.

Reply:
https://github.com/istio/istio.io/pull/4492